### PR TITLE
clean up the code in FormCompany.jsx

### PR DESCRIPTION
**Pull request by gaylnabil**

### CODE REVIEW

File: frontend/src/components/products/FormCompany.jsx [hunk 1]
```diff
@@ -8,17 +8,6 @@ function FormCompany() {
         name: "",
     });
 
-    // const handleValueChange = (event) => {
```
Comment (ngsakazaki) [inline frontend/src/components/products/FormCompany.jsx:11]: I delete this useless comment

**Review by ngsakazaki — APPROVED** [review]
it's ready for merging

File: frontend/src/components/products/FormCompany.jsx (hunk 1 above)
Comment (gaylnabil) [reply frontend/src/components/products/FormCompany.jsx:11]: OK. good

**Review by gaylnabil — COMMENTED** [review]
ok great, everything is reviewed

**Pull request merged**